src/components/SystemIntakeReview/index.tsx

```diff
@@ -0,0 +1,183 @@
+import { DateTime } from 'luxon';
+import React from 'react';
+import {
+  DescriptionDefinition,
+  DescriptionList,
+  DescriptionTerm
+} from '../shared/DescriptionGroup';
+import convertBoolToYesNo from '../../utils/convertBoolToYesNo';
+import { SystemIntakeForm } from '../../types/systemIntake';
+
+type SystemIntakeReview = {
+  heading: string;
+  systemIntake: SystemIntakeForm;
+  id?: string;
+};
+
+export const SystemIntakeReview = ({
+  heading,
+  systemIntake,
+  id
+}: SystemIntakeReview) => {
+  const fundingDefinition = () => {
+    const isFunded = convertBoolToYesNo(systemIntake.fundingSource.isFunded);
+    if (systemIntake.fundingSource.isFunded) {
+      return `${isFunded}, ${systemIntake.fundingSource.fundingNumber}`;
+    }
+    return isFunded;
+  };
+  const issoDefinition = () => {
+    const hasIsso = convertBoolToYesNo(systemIntake.isso.isPresent);
+    if (systemIntake.isso.isPresent) {
+      return `${hasIsso}, ${systemIntake.isso.name}`;
+    }
+    return hasIsso;
+  };
+  return (
+    <div className="system-intake__review margin-bottom-7">
+      <h1 className="font-heading-xl margin-top-4">{heading}</h1>
+      {!systemIntake && id && (
+        <h2 className="font-heading-xl">
+          System intake with ID: {id} was not found
+        </h2>
+      )}
+      {systemIntake && (
+        <div>
```

Comment:
In my opinion, the `SystemIntakeReview` should only be the presentation of the data filled out in the form. The heading and conditional logic to display the form data should be handled from with each individual use case.